disttask/framework/scheduler/manager.go

```diff
@@ -120,8 +124,24 @@ func (b *ManagerBuilder) BuildManager(ctx context.Context, id string, taskTable
 }
 
 // Start starts the Manager.
-func (m *Manager) Start() {
+func (m *Manager) Start() error {
 	logutil.Logger(m.logCtx).Debug("manager start")
+	var err error
+	for i := 0; i < retrySQLTimes; i++ {
```

Comment:
the variable.DefTiDBDDLErrorCountLimit default is 512, I think it is too big. I think retry 20 times, interval 0.5s. Then if 10s can not start then failed.

Reply:
> the variable.DefTiDBDDLErrorCountLimit default is 512, I think it is too big. I think retry 20 times or 10 times less than variable.DefTiDBDDLErrorCountLimit, interval 0.5s. Then if 10 to 20s can not start then failed.

This can be modified later since this variable also used in dispatcher, I think we need one pr to change it.